_episodes/05-mplhep.md

```diff
@@ -45,29 +45,21 @@ Now we will get some data stored in a github repository.
 
 ```python
 # Data for later use.
-csvs = [
-    pd.read_csv(
-        "https://raw.githubusercontent.com/GuillermoFidalgo/Python-for-STEM-Teachers-Workshop/master/data/4e_2011.csv"
-    ),
-    pd.read_csv(
-        "https://raw.githubusercontent.com/GuillermoFidalgo/Python-for-STEM-Teachers-Workshop/master/data/4mu_2011.csv"
-    ),
-    pd.read_csv(
-        "https://raw.githubusercontent.com/GuillermoFidalgo/Python-for-STEM-Teachers-Workshop/master/data/2e2mu_2011.csv"
-    ),
+file_names = [
+    "4e_2011.csv",
+    "4mu_2011.csv",
+    "2e2mu_2011.csv",
+    "4mu_2012.csv",
+    "4e_2012.csv",
+    "2e2mu_2012.csv",
 ]
-csvs += [
-    pd.read_csv(
-        "https://raw.githubusercontent.com/GuillermoFidalgo/Python-for-STEM-Teachers-Workshop/master/data/4mu_2012.csv"
-    ),
-    pd.read_csv(
-        "https://raw.githubusercontent.com/GuillermoFidalgo/Python-for-STEM-Teachers-Workshop/master/data/4e_2012.csv"
-    ),
+
+csvs = [
     pd.read_csv(
-        "https://raw.githubusercontent.com/GuillermoFidalgo/Python-for-STEM-Teachers-Workshop/master/data/2e2mu_2012.csv"
-    ),
+        f"https://raw.githubusercontent.com/GuillermoFidalgo/Python-for-STEM-Teachers-Workshop/master/data/{file_name}"
```

Comment:
could you perhaps first do a `basepath = ` and then `read_csv(f"{basepath}/{file_name}")` (just to keep lines shorter)?